### ISSUE TITLE
Weird Build Issue

### ISSUE THREAD

**Issue by ajnozari** [atyenoria/janus-webrtc-gateway-docker#52]
If you try to build with the following

`RUN sed -i 's/archive.ubuntu.com/mirror.aarnet.edu.au\/pub\/ubuntu\/archive/g' /etc/apt/sources.list

RUN rm -rf /var/lib/apt/lists/*`

the http rest api is disabled as libmicrohttpd isn't available.  It seems that something is going wrong with the mirror here, however eliminating those two lines still results in a working build.

**Comment by mattnelsonuk** [comment]
I had the same issue. Removing the line sucessfulyl builds but the REST server si disabled. This fixed it for me

```
#Bump some version specifically to work with Janus
RUN echo deb http://deb.debian.org/debian testing main >> /etc/apt/sources.list && \
	apt-get update && \
	apt-cache policy libmicrohttpd-dev

RUN apt-get install -y libmicrohttpd-dev=0.9.66-1+b1 
```

**Comment by metacoma** [comment]
@atyenoria I suppose, this issue it can be closed too according to #61
